tests/src/test/scala/whisk/core/apigw/actions/test/ApiGwRoutemgmtActionTests.scala

```diff
@@ -342,7 +212,7 @@ class ApiGwRoutemgmtActionTests
         return matches
     }
 
-    behavior of "API Gateway routemgmt action parameter validation"
```

Comment:
I am pretty sure that is how I implemented this change. Github is picking up the deletion and removal of V2 tags as a modify. Threw me off at first as well